core-bundle/contao/library/Contao/Widget.php

```diff
@@ -1403,9 +1403,9 @@ public static function getAttributesFromDca($arrData, $strName, $varValue=null,
 				$arrAttributes['maxlength'] = $arrAttributes['sql']['length'];
 			}
 
-			if (!isset($arrAttributes['unique']) && isset($arrAttributes['sql']['customSchemaOptions']['unique']))
+			if (!isset($arrAttributes['unique']) && (isset($arrAttributes['sql']['customSchemaOptions']['unique']) || isset($arrAttributes['sql']['platformOptions']['unique'])))
 			{
-				$arrAttributes['unique'] = $arrAttributes['sql']['customSchemaOptions']['unique'];
+				$arrAttributes['unique'] = $arrAttributes['sql']['customSchemaOptions']['unique'] ?? $arrAttributes['sql']['platformOptions']['unique'];
```

Comment:
```suggestion
				$arrAttributes['unique'] = $arrAttributes['sql']['platformOptions']['unique'] ?? $arrAttributes['sql']['customSchemaOptions']['unique'];
```

Shouldn't the new option have priority here?